cmd/dcrdata/views/charts.tmpl

```diff
@@ -189,7 +189,7 @@
         </div>
 
 
-        <div data-charts-target="chartWrapper" class="chart-wrapper ps-2 pe-2 mb-5">
+        <div data-charts-target="chartWrapper" class="chart-wrapper px-2 pb-5">
```

Comment:
OK!  Looks consistent with `padding-bottom`

Reply:
Yeah, just like we do for the `main` div in other pages.